views/ctltools/forms/delete-form.hbs

```diff
@@ -18,17 +18,17 @@
         <br>
         <label class="control-label" for="course">
             <input type="text" class="number-input-field" name="contextExternalToolCountPerContext[course]" id="course">
-            Kurs(e)
+            Kurs-Tools
```

Comment:
I asked the question if this dialog should display things like "2 Medienregal" or "1 Bereiche" and UX responded with: "Für mich würde es mehr Sinn machen, wenn wir das rumdrehen, also auf die Frage, wo wird welches Tool wie oft verwendet: Medienregal (2), Bereiche (1)"